pages/common/elvish.md

```diff
@@ -0,0 +1,20 @@
+# elvish
+
+> Expressive programming language and a versatile interactive shell.
+> More information: <https://elv.sh/learn/>.
```

Comment:
I prefer this link, because it shows you more general information and quickly navigates you to <https://elv.sh/learn/> if you want to.
 ```suggestion
> More information: <https://elv.sh>.
```

Reply:
What about to update other tldr pages to provide links to official tutorials but docs directly?